ha_healthchecker/ha_healthchecker/action/refresher.py

```diff
@@ -18,10 +18,13 @@ def _refresh_service(self, service_obj, node_obj):
         cur_status = self._get_service_status(service_obj.name)
         update_dict = {}
         if cur_status == 'up':
-            if service_obj.status != 'up':
+            if (service_obj.status != 'up' and
+                    service_obj.restarted_count <
+                    self.cluster_config.service_count_allow_to_raise):
```

Comment:
I think no matter what's the restarted_count  value, once the service is up, it should be set to 0, so just `if (service_obj.status != 'up'` is enough.

Reply:
Oh, yes.